### DISCUSSION TITLE
post_partner wrongly showed as primary amenity

### DISCUSSION THREAD

**Discussion by Michael21011** [osmandapp/OsmAnd#16708] [Maps Related]
There is specific value for post_office
post_office=post_partner
[Tag on wiki](https://wiki.openstreetmap.org/wiki/Tag:post_office%3Dpost_partner) 
It should be additional service for shop or bank
`In some countries it is possible, that there are POIs like shop=*, amenity=*, craft=* or office=* where the operator offers postal services on behalf. `
But OsmAnd shows it as primary value of POI.
For example check  shop=houseware 
![Screenshot_2023-03-15-11-21-39-944-edit_net osmand plus](https://user-images.githubusercontent.com/9266780/225251657-31b6b704-5e42-493f-973f-be55d921332e.jpg)
OsmAnd shows it as post office and hide any mentions about houseware shop.

OsmAnd~ 4.3.0#34773m, released: 2022-10-22

**Comment by Michael21011** [comment]
Example of shop https://www.openstreetmap.org/node/9293407664#map=18/53.46687/27.19803

**Comment by frasty** [comment]
Maybe this is related to this: https://github.com/osmandapp/OsmAnd/issues/16186#issue-1524194285
